src/Nethermind/Nethermind.HealthChecks/NodeHealthService.cs

```diff
@@ -24,16 +23,16 @@ public class CheckHealthResult
         public IEnumerable<string> Errors { get; set; }
     }
 
-    public class NodeHealthService : INodeHealthService
+    public class NodeHealthService : INodeHealthService, IAsyncDisposable
```

Comment:
Let's remove the implementation of `IAsyncDisposable`:

- Since we're injecting `IClHealthTracker` in the constructor, this class is not responsible for managing the lifetime of the dependency.
- The place where we inject a concrete implementation is `HealthChecksPlugin.InitRpcModules`, and that class already disposes of the implementation, so right now we're disposing twice. In this case it's fine sine we're essentially disposing a `Timer` twice.